test/stripe/topup_test.rb

```diff
@@ -0,0 +1,43 @@
+require File.expand_path("../../test_helper", __FILE__)
+
+module Stripe
+  class TopupTest < Test::Unit::TestCase
+    should "be listable" do
+      topups = Stripe::Topup.list
+      assert_requested :get, "#{Stripe.api_base}/v1/topups"
+      assert topups.data.is_a?(Array)
+      assert topups.data[0].is_a?(Stripe::Topup)
+    end
+
+    should "be retrievable" do
+      topup = Stripe::Topup.retrieve("tu_123")
+      assert_requested :get, "#{Stripe.api_base}/v1/topups/tu_123"
+      assert topup.is_a?(Stripe::Topup)
+    end
+
+    should "be creatable" do
+      topup = Stripe::Topup.create(
+        amount: 100,
+        currency: "USD",
+        source: "src_123",
+        description: "description",
+        statement_descriptor: "statement descriptor"
+      )
+      assert_requested :post, "#{Stripe.api_base}/v1/topups"
+      assert topup.is_a?(Stripe::Topup)
```

Comment:
For some reason this assertions, and the others like it, are failing because `topup` is a `Stripe::StripeObject` instead of a `Stripe::Topup`... I've looked at other similar resources and don't quite see what I've missed...?